addons/website_sale_product_configurator/views/website_sale_product_configurator_templates.xml

```diff
@@ -4,6 +4,21 @@
         <xpath expr="//th[hasclass('td-qty')]/span[hasclass('label')]" position="attributes">
             <attribute name='t-if'>not request.is_frontend or (request.is_frontend and is_view_active('website_sale.product_quantity'))</attribute>
         </xpath>
+        <xpath expr="//td[hasclass('td-price')]" position="inside">
```

Comment:
This puts the striked price after the "real" price. Could you put it before instead? (same below)

Reply:
Done :+1: